src/popup.html

```diff
@@ -83,13 +83,21 @@ <h4 class="i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></h4>
         <!-- Panel: Masks (Default once logged in) -->
         <masks-panel data-panel-id="masks" class="is-hidden fx-relay-panel" id="masks-panel">
           <div class="fx-relay-panel-content">
+            <!-- CSAT Survey -->
+            <div class="fx-relay-csat-survey-link-container is-hidden" role="alert"> 
+              <span class="i18n-content fx-relay-csat-survey-link" data-i18n-message-id="popupSurveyLink"></span>
```

Comment:
suggestion: make this a `<button>` or add a `tabindex`/`aria-role` to make it accessible by keyboard.

Reply:
cc/ @Vinnl

Reply:
Yeah, a `<button>` would handle a whole bunch of things (and `tabindex`/`role` wouldn't). Think activation on space/enter (one of these is `onKeyUp` and the other `onKeyDown` I believe), proper announcements by screen readers, and probably more things I don't even _want_ to have to think about :P 

(Since this is called `-link` my first thought was an `<a>`, but I don't think it's actually a link, i.e. it doesn't take the user anywhere.)

Reply:
I misunderstood, sorry! You want the span to also be a button - done.